common/djangoapps/student/models.py

```diff
@@ -1320,7 +1312,7 @@ def username(self):
 
     @property
     def course(self):
-        return modulestore().get_course(self.course_id)
+        return CourseOverview.get_from_id(self.course_id)
 
```

Comment:
By changing this to return a CourseOverview, this method is now a duplicate of `CourseEnrollment.course_overview`. Consider getting rid of the `.course` method entirely, and changing everything to call `.course_overview`?